src/drawing.cpp

```diff
@@ -273,7 +273,7 @@ void ShowStrip(uint16_t numToShow)
 
 // CalcDelayUntilNextFrame
 //
-// Returns the amount of time to wait patiently until its time to draw the next frame, up to one second max
+// Returns the amount of time to wait patiently until it is time to draw the next frame, up to one second max

```

Comment:
Same comment as before.

```suggestion
// Returns the amount of time to wait patiently until it's time to draw the next frame, up to one second max
```